app/src/main/res/layout/item_recent_call.xml

```diff
@@ -97,20 +97,48 @@
         tools:text="4 July, 04:37 PM"
         tools:textColor="@color/color_missed_call" />
 
+    <TextView
+        android:id="@+id/item_recents_date_time_duration_separator"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/item_recents_name"
+        android:alpha="0.6"
+        android:gravity="end"
+        android:textSize="@dimen/normal_text_size"
+        app:layout_constraintBottom_toBottomOf="@id/item_recents_date_time"
+        app:layout_constraintStart_toEndOf="@+id/item_recents_date_time"
+        app:layout_constraintTop_toTopOf="@id/item_recents_date_time"
+        tools:text=" • " />
+
     <TextView
         android:id="@+id/item_recents_duration"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_below="@+id/item_recents_name"
-        android:layout_marginEnd="@dimen/small_margin"
         android:alpha="0.6"
         android:gravity="end"
         android:textSize="@dimen/normal_text_size"
         app:layout_constraintBottom_toBottomOf="@id/item_recents_date_time"
-        app:layout_constraintEnd_toStartOf="@+id/overflow_menu_icon"
+        app:layout_constraintStart_toEndOf="@id/item_recents_date_time_duration_separator"
         app:layout_constraintTop_toTopOf="@id/item_recents_date_time"
         tools:text="00:10" />
 
+    <TextView
+        android:id="@+id/item_recents_location"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:layout_below="@+id/item_recents_name"
```

Comment:
There should be some margin between duration and location, especially for places with longer names or for smaller screen sizes.

```suggestion
        android:layout_below="@+id/item_recents_name"
        android:layout_marginStart="@dimen/normal_margin"
        android:layout_marginEnd="@dimen/small_margin"
```

The end margin is just for aligning it with the title view.